src/compiler/program.ts

```diff
@@ -2702,6 +2707,34 @@ export function createProgram(rootNamesOrOptions: readonly string[] | CreateProg
         return hasEmitBlockingDiagnostics.has(toPath(emitFileName));
     }
 
+    function getCustomTransformers() {
+        if (!host.require) {
+            return emptyArray;
+        }
+
+        const compilerOptions = program.getCompilerOptions();
+        if (!shouldAllowPlugins(compilerOptions)) {
+            return emptyArray;
+        }
+
+        const customTransformers = mapDefined(compilerOptions.plugins, config => {
+            if (config.type !== "transformer") return undefined;
+
+            // TODO(jakebailey): The LS plugin loader is more complicated than this; copy.
+            const result = host.require!(program.getCurrentDirectory(), config.path);

```

Comment:
I think it must be start from location of tsconfig.json file, you're using current-working-directory.

Reply:
This is theoretically the tsconfig dir, but yes, this code is very WIP.

Reply:
Hmm. Oops, but all of this is going to be replaced anyway. This branch is effectively demo code at the moment.